### DISCUSSION TITLE
Hide route at certain high zoom, only keep arrows

### DISCUSSION THREAD

**Discussion by an-dr-eas-k** [osmandapp/OsmAnd#11790] [Ideas]
When using touring view for hiking I depend on the track type that I am supposed to follow during navigation.

I would like to have the feature, the blue line to disappear when zooming in. The arrows may stay. Currently the blue route hides the track type and there is no way to see, weather the route is acceptable for kids.

starting to code that myself, but no success. didnt get the switch, case, apply_if right...

**Comment by an-dr-eas-k** [comment]
This is a version that is not working. Higher zoom levels don't change the route color. What can I change to achieve that?
`<?xml version="1.0" encoding="utf-8"?>
<renderingStyle name="hide navroute at high zoom" depends="Touring-view_(more-contrast-and-details)" version="1">
	<renderingAttribute name="route">
<case strokeWidth="10:6" color_3="#2EFF00" strokeWidth_3="5:3">
<apply maxzoom="14" color="#88aaaaaa" />
<apply_if minzoom="15" color="#ffffffff" /> 
</case>
      
</renderingAttribute>
</renderingStyle>`

**Comment by vshcherb** [comment]
I actually think it's going to be fixed in tomorrow nightlies @cepprice . There is a bug with rendering a route. Actually route coloring by steepness is not part of rendering style.

**Comment by an-dr-eas-k** [comment]
is the PR #11868 helping here? how can I use the nightly builds with my purchases (contour-lines)?
